include/soci/soci-platform.h

```diff
@@ -95,6 +95,12 @@ namespace std {
 // mode, we just need to check for the minimal compiler version supporting them
 // (see https://msdn.microsoft.com/en-us/library/hh567368.aspx).
 
+#if defined(SOCI_HAVE_CXX_C11) || (defined(_MSC_VER) && _MSC_VER >= 1800)
```

Comment:
Again, hardly important, but it is actually supposed to work with 1700 (VC11) too.

Reply:
Yes, it should even, partially or perhaps work with MSVC 1600 as per [Support For C++11/14/17 Features](https://msdn.microsoft.com/en-us/library/hh567368.aspx).